### ISSUE TITLE
Second run of setup.sh breaks everything

### ISSUE THREAD

**Issue by failinghobbyist** [rajannpatel/Pi-Hole-on-Google-Compute-Engine-Free-Tier-with-Full-Tunnel-and-Split-Tunnel-Wireguard-VPN-Configs#44]
Thanks for the guide! Running through it once and generating the first key is very smooth. Using a phone with the first QR code works, and modding to go full tunnel works fine. 

As soon as I run setup for a second time to generate another key, the first client dies. It appears to connect, but can't reach either the pi hole admin console or the internet. 

What am I doing wrong? I notice during the second setup run that its trying to assign the same client IP again (10.66.66.2) which seems like a problem. 

Is there a way to blank the wg config so that I don't have to spin up a new VM every time I start over?

Thanks again for the help, enjoyed learning through your guide!

**Comment by chicocheco** [comment]
Are you sure that the qr code you are scanning is not the same as the first one?

**Comment by LloydGSR** [comment]
I had the same issue.  Generating a second profile was overwriting the first and would make the QR code substantially larger, increasing in size each time a client was added.

To get around it, when running the script again, it'd offer 10.66.66.2 again, so I just removed the 2 and typed 3.  I did the same for the IPv6 address.

The resulting profile had both the .2 and the .3 addresses for the client, so I just removed the .2 stuff and it worked.  I very much doubt this is a proper way of doing it, but it worked.
